lazyflow/operators/cacheMemoryManager.py

```diff
@@ -280,3 +275,28 @@ def enable(self):
             self._disabled = False
         with self._condition:
             self._condition.notifyAll()
+
+
+_cache_memory_manager = _CacheMemoryManager()
+totalCacheMemory = _cache_memory_manager.totalCacheMemory
+
+
+def CacheMemoryManager():
```

Comment:
Tests seem to be passing, but are you sure no one is doing stuffthat relies on CacheMemoryManager being a class like:
`CacheMemoryManager.__class__`
or
```python3
class MyOtherCacheManager(CacheMemoryManager):
    ...
```
?

I've found these log configurations in `ilastik/ilastik/ilastik_logging/default_config.py:156 `:

`"lazyflow.operators.cacheMemoryManager.CacheMemoryManager": {"level": "INFO"},`

and similar in `ilastik/ilastik/ilastik_logging/logging_config.json`. Do you know if they will still work if CachememoryManager is no longer a class?

Reply:
so just for the logging, we can safely remove the trailing `.CacheMemoryManager`. ilastik/ilastik of course needs some changes, too. But I haven't seen any subclassing or things like that.

Reply:
Even now there is no logger with this name :) Only existing logger is "lazyflow.operators.cacheMemoryManager"

Reply:
> Tests seem to be passing, but are you sure no one is doing stuffthat relies on CacheMemoryManager being a class like:
> `CacheMemoryManager.__class__`
> or
> 
> ```python
> class MyOtherCacheManager(CacheMemoryManager):
>     ...
> ```
Of this I am sure, run tests in ilastik and inspected the code in volumina it's not used at all